ENCODE_release.py

```diff
@@ -426,17 +489,20 @@ def run_script(self):
         print("Releasenator version " + str(self.releasenator_version))
         for accession in self.ACCESSIONS:
             print("Processing accession: " + accession)
-            expandedDict = encodedcc.get_ENCODE(accession, self.connection)
-            objectStatus = expandedDict.get("status")
-            obj = expandedDict["@type"][0]
-
+            data = encodedcc.get_ENCODE(accession, self.connection)
+            data_status = data.get('status')
+            data_type = data['@type'][0]
+            if not self.HELA:
```

Comment:
This can be one condition so long as self.HELA is on the left of the AND.   If the first condition fails then python will not check the second one.